tiledb/cf/creator.py

```diff
@@ -717,6 +579,18 @@ class ArrayCreator:
              coordinate. Only allowed for sparse arrays.
     """
 
+    __slots__ = (
```

Comment:
Why adding slots in this class specifically? [`__slots__`](https://stackoverflow.com/a/28059785) is usually a access/memory (micro)optimization, which is probably not needed here.